docs/USAGE.md

```diff
@@ -0,0 +1,282 @@
+| `OTEL_MAX_LOGFILE_SIZE` | `10 MB` | The maximum size for tracer log files, in bytes. |
+| `OTEL_TRACE_LOG_PATH` | Linux: `/var/log/OTEL/dotnet/dotnet-profiler.log`<br>Windows: `%ProgramData%"\OTEL .NET Tracing\logs\dotnet-profiler.log` | The path of the profiler log file. |
+| `OTEL_DIAGNOSTIC_SOURCE_ENABLED` | `true` | Enable to generate troubleshooting logs with the `System.Diagnostics.DiagnosticSource` class. |
+| `OTEL_DISABLED_INTEGRATIONS` |  | The integrations you want to disable, if any, separated by a semi-colon. These are the supported integrations: AspNetMvc, AspNetWebApi2, DbCommand, ElasticsearchNet5, ElasticsearchNet6, GraphQL, HttpMessageHandler, IDbCommand, MongoDb, NpgsqlCommand, OpenTracing, ServiceStackRedis, SqlCommand, StackExchangeRedis, Wcf, WebRequest |
+| `OTEL_TRACE_DOMAIN_NEUTRAL_INSTRUMENTATION` | `false` |  Sets whether to intercept method calls when the caller method is inside a domain-neutral assembly. This is recommended when instrumenting IIS applications. |
+| `OTEL_PROFILER_PROCESSES` |  | Sets the filename of executables the profiler can attach to. If not defined (default), the profiler will attach to any process. Supports multiple values separated with semi-colons, for example: `MyApp.exe;dotnet.exe` |
+| `OTEL_PROFILER_EXCLUDE_PROCESSES` |  | Sets the filename of executables the profiler cannot attach to. If not defined (default), the profiler will attach to any process. Supports multiple values separated with semi-colons, for example: `MyApp.exe;dotnet.exe` |
+| `OTEL_CONVENTION` | `Datadog` | Sets the outbound http and trace id convention for tracer. Available values are: `Datadog` (64bit trace id), `OpenTelemetry` (128 bit trace id). |
+| `OTEL_PROPAGATOR` | `Datadog` | Sets the propagator for tracer. Available values are: `Datadog`, `B3`, `W3C`. |
+
+### Linux
+
+After downloading the library, install the CLR Profiler and its components
+via your system's package manager.
+
+1. Download the latest release of the library.
+2. Install the CLR Profiler and its components with your system's package
+manager:
+    ```bash
+    # Use dpkg:
+    $ dpkg -i otel-dotnet-autoinstrumentation.deb
+
+    # Use rpm:
+    $ rpm -ivh otel-dotnet-autoinstrumentation.rpm
+
+    # Install directly from the release bundle:
+    $ tar -xf otel-dotnet-autoinstrumentation.tar.gz -C /
+
+    # Install directly from the release bundle for musl-using systems (Alpine Linux):
+    $ tar -xf otel-dotnet-autoinstrumentation-musl.tar.gz -C /
+    ```
+1. Configure the required environment variables:
+    ```bash
+    $ source /opt/otel-dotnet-autoinstrumentation/defaults.env
+    ```
+2. Set the service name:
+    ```bash
+    $ export OTEL_SERVICE_NAME='MyCoreService'
+    ```
+3. Set Zipkin exporter:
+    ```bash
+    $ export OTEL_EXPORTER='Zipkin'
+    ```
+4. Set OpenTelemetry conventions:
+    ```bash
+    $ export OTEL_CONVENTION='OpenTelemetry'
+    ```
+5. Set the endpoint of a Smart Agent or OpenTelemetry Collector:
```

Comment:
Do not use Smart Agent. In this case something like "Set the endpoint, e.g. OpenTelemetry Collector:"